<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyEditClass</class>
 <widget class="QMainWindow" name="FlyEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlyEdit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Submit">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>326</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>提交修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ReMod">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>276</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>撤销修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleRow">
    <property name="geometry">
     <rect>
      <x>583</x>
      <y>406</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除选中行</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>206</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_Down">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>当前行下</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Up">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>当前行上</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_InsRow">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>12</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>插入行</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_ImportExcel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>从Excel导入新数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ExportToExcel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>将数据导出到Excel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>参数列表</string>
    </property>
    <widget class="QTreeView" name="treeView_paraList">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>21</y>
       <width>189</width>
       <height>509</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>291</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>所选参数数据信息</string>
    </property>
    <widget class="QTableView" name="tableView_Para1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>511</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>pushButton_Submit</zorder>
   <zorder>pushButton_ReMod</zorder>
   <zorder>pushButton_DeleRow</zorder>
   <zorder>groupBox</zorder>
   <zorder>pushButton_ImportExcel</zorder>
   <zorder>pushButton_ExportToExcel</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="flyedit.qrc"/>
 </resources>
 <connections/>
</ui>
